sympy/assumptions/sathandlers.py

```diff
@@ -228,6 +228,15 @@ def evaluate_old_assump(pred):
     """
     return pred.xreplace(Transform(_old_assump_replacer))
 
+class ListAssump():
```

Comment:
Because of: https://github.com/sympy/sympy/issues/11587 It might be easier for users to use this function directly.

Also, it can be useful to keep it as a function to expand it's usecases at some point of time.